docs/DeveloperGuide.md

```diff
@@ -150,16 +150,16 @@ API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/
 
 The `Model` component,
 
-* stores the address book data i.e., all `Person` objects (which are contained in a `UniquePersonList` object).
-* stores the currently 'selected' `Person` objects (e.g., results of a search query) as a separate _filtered_ list which
-  is exposed to outsiders as an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to
+* stores the application data i.e., all `Entry` objects (which are contained in a `UniqueEntryList` object).
+* stores the currently 'selected' `Entry` objects (e.g., results of a search query) as a separate _filtered_ list which
+  is exposed to outsiders as an unmodifiable `ObservableList<Entry>` that can be 'observed' e.g. the UI can be bound to
   this list so that the UI automatically updates when the data in the list change.
 * stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as
   a `ReadOnlyUserPref` objects.
 * does not depend on any of the other three components (as the `Model` represents data entities of the domain, they
   should make sense on their own without depending on other components)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
+<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `PennyWise`, which `Entry` references. This allows `PennyWise` to only require one `Tag` object per unique tag, instead of each `Entry` needing their own `Tag` objects.<br>
```

Comment:
this might be misleading as we don't have a tag list as we took away the option of having multiple tags

Reply:
Okay, ill remove this entirely